avaframe/com1DFA/com1DFA.py

```diff
@@ -1582,6 +1586,53 @@ def initializeMassEnt(dem, simTypeActual, entLine, reportAreaInfo, thresholdPoin
 
     return entrMassRaster, entrEnthRaster, reportAreaInfo
 
+def initializeDetrainment(cfg, dem, simTypeActual, resLine, reportAreaInfo, thresholdPointInPoly):
```

Comment:
is this needed or already done in the initializeResistance? is the initialzeDetrainment called somewhere? for what flags?

Reply:
not needed anymore -> I've deleted it